src/sharedInterfaces/schemaDesigner.ts

```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface ISchema {
+    entities: IEntity[];
+    relationships: IRelationship[];
+}
+
+export interface IEntity {
+    /**
+     * Name of the entity
+     */
+    name: string;
```

Comment:
for entities we should consider where to use a URI vs. schema/name?  Like if we wanted to include multiple DBs in the future, or non-table types.  This is probably fine, but something to think about.  We had this problem with the Metadata lookup stuff in ADS, and later wished we had used SFI Paths or something like that for more flexible/deterministic entity referencing.